app/stac_api/serializers.py

```diff
@@ -514,23 +515,37 @@ class Meta:
 
     # NOTE: when explicitely declaring fields, we need to add the validation as for the field
     # in model !
+    item = serializers.SlugRelatedField(
+        slug_field='name', write_only=True, queryset=Item.objects.all()
+    )
+    id = serializers.CharField(
+        source='name',
+        required=True,
+        max_length=255,
+        validators=[validate_name, UniqueValidator(queryset=Asset.objects.all())]
+    )
+    title = serializers.CharField(max_length=255, required=False, allow_blank=True)
     type = serializers.CharField(source='media_type', max_length=200)
     # Here we need to explicitely define these fields with the source, because they are renamed
     # in the get_fields() method
+    description = serializers.CharField(max_length=255, required=False, allow_blank=True)
     eo_gsd = serializers.FloatField(source='eo_gsd', required=False, allow_null=True)
     geoadmin_lang = serializers.ChoiceField(
         source='geoadmin_lang',
-        choices=['de', 'fr', 'it', 'rm', 'en'],
+        choices=['de', 'fr', 'it', 'rm', 'en', ''],
```

Comment:
why an empty string here?

Reply:
it's a valid choice. But maybe use `Language.NONE` instead (in the rare case this value will change)

Reply:
Good point, how about `choices = Asset.Language.values` here? This way we'd always be consistent with the definition in the models. For the "empty language" value as well as all the other languages?

Reply:
even better ;-)